src/main/java/data_access/ApiDataAccessObject.java

```diff
@@ -121,4 +123,5 @@ private JSONObject getResponseBody(OkHttpClient client, Request request) {
 
         return responseBody;
     }
+
```

Comment:
Was this space added by accident?

Reply:
I did not touch this class, maybe Amber added this? I'll ask her

Reply:
She didn't touch it either. 
![image](https://github.com/user-attachments/assets/38bf96e8-9e56-47d5-b7f8-5f15c9a7f452)
This is what shows up if I remove the return statement.
So I'm gonna leave it there